conf/application.conf

```diff
@@ -171,7 +171,7 @@ allowedList {
 
 features {
     returnsEnabled = true
-    creditsForReturnsEnabled = true
+    creditsForReturnsEnabled = false
```

Comment:
The ATs now have an expectation that this is `true` - Was this supposed to be set to False?